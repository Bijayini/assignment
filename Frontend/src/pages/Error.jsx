import React from "react";
import PropTypes from "prop-types";

import labels from "../labels/Error.labels";
import '../styles/error.scss';

const Error = ({message}) => {
    return <div className="wrap error-page">
        <h1 className="error-title">{message}</h1>
        <p>{labels.message}</p>
    </div>;
};

Error.defaultProps = {
    message:labels.something_went_wrong
};

Error.propTypes = {
    message: PropTypes.string,
};


export default Error;