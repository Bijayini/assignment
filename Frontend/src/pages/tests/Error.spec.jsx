import React from 'react';
import { shallow } from 'enzyme';

import Error from '../Error';

describe('<Error />', () => {
  let component;

  beforeEach(() => {
    component = shallow(<Error />);
  });

  it('should match with snapshot', () => {
    expect(component.debug()).toMatchSnapshot();
  });
});