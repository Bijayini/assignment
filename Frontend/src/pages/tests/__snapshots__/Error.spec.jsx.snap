// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Error /> should match with snapshot 1`] = `
"<div className=\\"wrap error-page\\">
  <h1 className=\\"error-title\\">
    Something Went Wrong !!
  </h1>
  <p>
    Please join us after sometime
  </p>
</div>"
`;
